deploy/ansible/hosts/feature.yml

```diff
@@ -8,22 +8,20 @@ cores:
       hosts:
         node-01.feature.shimmer.iota.cafe:
           validatorAccount: "{{ NODE_01_ACCOUNTID }}"
-          validatorPrivKey: "{{ NODE_01_VALIDATOR_PRIVKEY }}"
-          p2pIdentityPrivateKey: "{{ NODE_01_P2PIDENTITYPRIVATEKEY }}"
+          validatorPrvKey: "{{ NODE_01_VALIDATOR_PRIVKEY }}"
+          p2pIdentityPrvKey: "{{ NODE_01_P2PIDENTITYPRIVATEKEY }}"
         node-02.feature.shimmer.iota.cafe:
           validatorAccount: "{{ NODE_02_ACCOUNTID }}"
-          validatorPrivKey: "{{ NODE_02_VALIDATOR_PRIVKEY }}"
-          p2pIdentityPrivateKey: "{{ NODE_02_P2PIDENTITYPRIVATEKEY }}"
+          validatorPrvKey: "{{ NODE_02_VALIDATOR_PRIVKEY }}"
+          p2pIdentityPrvKey: "{{ NODE_02_P2PIDENTITYPRIVATEKEY }}"
         node-03.feature.shimmer.iota.cafe:
           validatorAccount: "{{ NODE_03_ACCOUNTID }}"
-          validatorPrivKey: "{{ NODE_03_VALIDATOR_PRIVKEY }}"
-          p2pIdentityPrivateKey: "{{ NODE_03_P2PIDENTITYPRIVATEKEY }}"
+          validatorPrvKey: "{{ NODE_03_VALIDATOR_PRIVKEY }}"
+          p2pIdentityPrvKey: "{{ NODE_03_P2PIDENTITYPRIVATEKEY }}"
         node-04.feature.shimmer.iota.cafe:
-          validatorAccount: "{{ NODE_04_ACCOUNTID }}"
-          validatorPrivKey: "{{ NODE_04_VALIDATOR_PRIVKEY }}"
-          p2pIdentityPrivateKey: "{{ NODE_04_P2PIDENTITYPRIVATEKEY }}"
+          p2pIdentityPrvKey: "{{ NODE_04_P2PIDENTITYPRIVATEKEY }}"
+          blockissuerPrvKey: "{{ BLOCKISSUER_PRV_KEY }}"
```

Comment:
Can you use the same naming scheme for the new variables? Changing them in the environment is a bit of a PITA